packages/app-mobile/components/checkbox.tsx

```diff
@@ -1,45 +1,62 @@
-const React = require('react');
+import * as React from 'react';
 const Component = React.Component;
 const { View, TouchableHighlight } = require('react-native');
 const Icon = require('react-native-vector-icons/Ionicons').default;
 Icon.loadFont();
 
-const styles = {
+interface Style {
+	[key: string]: any;
+}
+interface Props {
+	style: Style;
+	checked: boolean;
+	onChange: (checked: boolean)=> void;
+	accessibilityLabel?: string;
+}
+
+interface State {
+	checked: boolean;
+}
+
+const styles: Style = {
 	checkboxIcon: {
 		fontSize: 20,
 		height: 22,
 		// marginRight: 10,
 	},
 };
 
-class Checkbox extends Component {
-	constructor() {
-		super();
+class Checkbox extends Component<Props, State> {
+	state: State;
+	props: Props;
```

Comment:
private

Reply:
Noted.

Reply:
It threw an error when I set it as private, but I was able to set the types like this 
```
class Checkbox extends Component<Props, State> {
```